gc/base/Configuration.hpp

```diff
@@ -178,11 +178,7 @@ class MM_Configuration : public MM_BaseVirtual
 	 * @param[in] env the current environment.
 	 * @return boolean indicating whether the configuration was successfully updated.
 	 */
-	virtual bool reinitializeForRestore(MM_EnvironmentBase* env)
-	{
-		Assert_MM_unreachable();
-		return false;
-	}
+	virtual bool reinitializeForRestore(MM_EnvironmentBase *env);
```

Comment:
To be consistent with the existing format of the file.
```suggestion
	virtual bool reinitializeForRestore(MM_EnvironmentBase* env);
```

Reply:
the first style is actually more popular in overall OMR (and J9) code, so next time we touch this file I'd actually prefer to convert them all back

Reply:
Yes, `int32_t *x` is preferred over `int32_t* x`. @RSalman I still see a mixture of both styles in the PR. Can you make the changes consistent?

Reply:
I've converted all the touched files to use `int32_t* x` style its a lot more used in the files